packages/account-management/src/frontend/AccountManagement/AccountManagement.tsx

```diff
@@ -6,6 +6,7 @@ import SpacingComponent from '@weco/common/views/components/SpacingComponent/Spa
 import { ProfileForm } from './ProfileForm';
 import { PasswordForm } from './PasswordForm';
 import { LogoContainer } from '../Shared/LogoContainer';
+import { PageWrapper } from '..//Shared/PageWrapper';
```

Comment:
Is there a typo in the path?

```suggestion
import { PageWrapper } from '../Shared/PageWrapper';
```